contracts/tgrade-valset/src/multitest/verify_online.rs

```diff
@@ -80,7 +81,7 @@ fn verify_validators_jailing() {
         "member2member2member2member2memb",
     ];
 
-    let mut suite = SuiteBuilder::new()
+    let mut suite: Suite = SuiteBuilder::new()
```

Comment:
nit: This is superflous.

Reply:
Indeed. But, it magically enables "Go To" (Cmd+Option+B) functionality in IntelliJ. :-D